tasks/db_tasks.py

```diff
@@ -69,21 +69,38 @@ def seed_dev_db(c):
 
         with app.app_context():
 
-            from tests.conftest import seed_database_randomly
+            from tests.conftest import seed_database_for_fund_round
             from config import Config
 
             choosing = True
 
             while choosing:
 
-                apps = int(_echo_input("How many applications per round?"))
-                rounds = int(_echo_input("How many rounds per fund?"))
-                funds = int(_echo_input("How many funds?"))
+                from fsd_utils import CommonConfig
+                from uuid import uuid4
+
+                config = {
+                    "COFR2W2": {
+                        "fund_id": CommonConfig.COF_FUND_ID,
+                        "round_id": CommonConfig.COF_ROUND_2_ID
+                    },
+                    "RANDOM_FUND_ROUND": {
+                        "fund_id": uuid4(),
+                        "round_id": uuid4()
+                    }
+                }
+
+                new_line = '\n'
+                _echo_print(
+                    f"fund-rounds available to seed: {new_line} - {f' {new_line} - '.join(config.keys())}",
+                )
+                fund_round_input = str(_echo_input("Please type the fund-round to seed:"))
+                fund_round = config[fund_round_input]
+                apps = int(_echo_input("How many applications?"))
 
                 choosing = (
                     not _echo_input(
-                        f"This will create {apps * rounds * funds} rows. Would"
-                        " you like to continue? y/n \n"
+                        f"Would you like to insert {apps} applications for {fund_round_input}? y/n \n"
```

Comment:
I think we can access the containers on paas using 'docker exec -it' ?

Reply:
Following our conversation, it may be possible to invoke the script interactively by ssh'ing onto the cf app but I have now pushed an extension to this function which allows script arguments to be passed, removing the need for interactiveness.